app/controllers/events_controller.rb

```diff
@@ -1,12 +1,21 @@
 class EventsController < ApplicationController
-  # GET /events or /events.json
+  # GET /events
   def index
     @events = Event.upcoming
   end
 
+  # GET /events/all or /events/all.ics
   def all
     @events = Event.all
-    render :index
+
+    respond_to do |format|
+      format.html { render :index }
+      format.ics { render plain: Calendar.new(@events).to_ical}
+    end
+  end
+
+  def show
+    @event = Event.find(params[:id])
```

Comment:
I know slugs are generally better for SEO purposes, is that the primary motivation for this suggestion? I personally don't have a preference either way, if you think slugs are a good idea, I'd prefer to just implement them without adding a dependency like FriendlyID.

Not a blocker for this PR, can be follow on (which I'm happy to do!)

Reply:
We don't need slugs necessarily, we just need a uniquely identifiable URL for the meetups page so we can match it again to detect changes/updates.

I just thought it would make sense now, so we don't have to deal with it at a later point.

Reply:
I implemented slugs using a virtual column, simple enough and does the trick 🙌🏼